src/server/game/Globals/ObjectMgr.h

```diff
@@ -1254,6 +1267,16 @@ class TC_GAME_API ObjectMgr
             return nullptr;
         }
 
+        SpawnData const* GetSpawnData(SpawnObjectType type, ObjectGuid::LowType guid)
+        {
+            if (type == SPAWN_TYPE_CREATURE)
+                return GetCreatureData(guid);
+            else if (type == SPAWN_TYPE_GAMEOBJECT)
+                return GetGameObjectData(guid);
+            else
+                ASSERT(false, "Invalid spawn object type %u", uint32(type));
```

Comment:
Errors.h is not included in this file but brought in by some other header and I don't like that this code abuses that (move to .cpp?)